devlib/module/cgroups.py

```diff
@@ -394,7 +394,7 @@ def __init__(self, target):
     def list_subsystems(self):
         subsystems = []
         for line in self.target.execute('{} cat /proc/cgroups'\
-                .format(self.target.busybox)).splitlines()[1:]:
+                .format(self.target.busybox), as_root=True).splitlines()[1:]:
```

Comment:
Please change to `as_root=self.target.is_rooted`, so that this will continue to work for unrooted targets that do not require root to read `/proc/cgroups`.